doc/source/release-notes/8.0.29-21.rst

```diff
@@ -0,0 +1,41 @@
+.. _8.0.29-21:
+
+================================================================================
+*Percona Server for MySQL* 8.0.29-21 (2022-07-)
+================================================================================
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 8.0.29-21
+includes all the features and bug fixes available in the
+`MySQL 8.0.29 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-29.html>`__
+in addition to enterprise-grade features developed by Percona.
+
+.. include:: ../_res/rn/ps-mysql-blurb.txt
+
+.. contents::
+   :local:
+
+Release Highlights
+=================================================
+
+Improvements and bug fixes introduced by Oracle for *MySQL* 8.0.29 and included in Percona Server for MySQL are the following:
+
+* The Performance Schema tracks if a query was processed on the PRIMARY engine, InnoDB, or a SECONDARY engine, HeatWave. An EXECUTION_ENGINE column, which indicates the engine used, was added to the Performance Schema statement event tables and the `sys.processlist` and the `sys.x$processlist` views.
+* Added support for the `IF NOT EXISTS` option for the `CREATE FUNCTION`, `CREATE PROCEDURE`, and `CREATE TRIGGER` statements.
+* Added support for `ALTER TABLE ... DROP COLUMN ALGORITHM=INSTANT`. 
+* An anonymous user with the `PROCESS` privilege was unable to select `processlist` table rows.
+
+Find the full list of bug fixes and changes in the `MySQL 8.0.29 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-29.html>`__.
```

Comment:
we also have to add a special note about INSTANT corruption and PS choosing INPLACE as default for ADD and DROP columns?